docs/source/environment.rst

```diff
@@ -2,150 +2,131 @@
 
 Environment
 ***********
-This page describes the environments in which Fab can be run.
+
+Fab requires a suitible Python environment in which to run. This page outlines
+some routes to achieving such an environment.
+
+This page contains general instructions, there are additional instructions for
+:ref:`Met Office<MetOffice>` users elsewhere.
 
 
 .. _Requirements:
 
 Requirements
 ============
+
 The minimum dependencies for Fab to build a project are:
 
 * Python >= 3.7
-* Compiler
-* Linker (or archiver, if building libraries)
+* fParser >= 0.16.0
+* Fortran compiler
+* Linker
 
-If your project includes C code:
+If you intend to compile C you will need a C compiler and linker but also:
 
 * libclang
 * python-clang
 
+If you want pretty charts to be plotted from the build metrics you will need:
 
-Running Fab at The Met Office
-=============================
+* matplotlib
 
-VDI
----
 
-Without clang
-^^^^^^^^^^^^^
-If you don't need to build C, you can use this modules command::
+Obtaining Fab
+=============
 
-    $ module use $EXTRA_SOFTWARE
-    $ module load fab/0.11.0/python/3.7.0
+Although you can install the tool from a copy of the `source`_ it is usually
+preferable to use one of the managed packages.
 
+Packages are made available through `PyPI`_ and `Conda forge`_ and are installed
+in the usual way for these services. Some examples follow.
 
-With clang
-^^^^^^^^^^
-This is currently a little awkward but will hopefully become simpler soon.
-We use a combination of *module* commands and *conda* to get Fab running at The Met Office.
-If you don't need to compile C code, you may not need conda.
-We create the conda environment as follows::
+.. _source: https://github.com/metomi/fab
+.. _PyPI: https://pypi.org/project/sci-fab/
+.. _Conda forge: https://anaconda.org/conda-forge/sci-fab
 
-    $ conda env create -f envs/conda/dev_env.yml
-    $ conda activate sci-fab
 
-Then :ref:`install fab<Install>`.
+Somebody Elses Problem
+----------------------
 
-We do this in a before any module commands.
-Then we set up our environment *in a new terminal* as follows.
+Some of the prerequisites may require system administrator intervention. If they
+are feeling benevolent your administrator may even be prevailed upon to install
+the whole lot.
```

Comment:
I like the title of this section (😄), does this mean that I can contact an admin to create the environment for me, or would they install all of Fab for me? Would this be a ServiceNow request for MO users?

Reply:
This is general documentation for all users. In the case of MO, no, there is no-one else who will do this for you.